### ISSUE TITLE
Scollbars for the Mediabox type: HTML

### ISSUE THREAD

**Issue by Soldier2000** [stefanwalther/sense-media-box#21]
Only the website MediaBox type has the option to enable the scrollbars, the HTML type doesn't have this.

Loading HTML pages makes only sense when there are scrollbars available that enable you to see the whole page.

**Comment by stefanwalther** [comment]
Hi @Soldier2000, I am working on it. Should be able to publish a new version within the next days. Comes along with some general updates, that's why it takes unfortunately a bit longer.

**Comment by stefanwalther** [comment]
Here we go, haven't merged yet, but feel free to test:
https://github.com/stefanwalther/sense-media-box/tree/html-scrollbars
Test with v0.4.5 which can be found at: https://github.com/stefanwalther/sense-media-box/tree/html-scrollbars/build

**Comment by Soldier2000** [comment]
Thanks, I tried both the builds, but i couldn't find the scroll bars

![html](https://user-images.githubusercontent.com/25410164/34933839-b3edd478-f9d8-11e7-8896-051cb80d0cb0.JPG)

**Comment by stefanwalther** [comment]
Did you check the properties and enable scrolling to either "always on" or "auto":

![image](https://user-images.githubusercontent.com/669728/34948041-01084e86-f9da-11e7-8f5b-fc89b949c28e.png)

**Comment by Soldier2000** [comment]
Aaah, that explains, i don't have that option here. 
I'm running version 11.14.3 (September) on my desktop, are you running an older or newer version?

**Comment by stefanwalther** [comment]
That's independent from the Qlik Sense version, that's a new functionality of the sense-media-box extension. Pls make sure that you uninstall the old version of your extension and clean the browser cache and install the new extension.
Have a look e.g. here:

- https://vimeo.com/207511145
or
- http://qliksite.io/tutorials/qliksense-visualization-extensions/part-09/1002-Troubleshooting-FAQ/#

**Comment by Soldier2000** [comment]
I've removed the extension, copy past the scrollbar extension and removed the cache. 
Now it seems that the scrollbar extension doesn't work at all, when I drag and drop the extension on my sheet i receive the message: Invalid Visualization. 

When I replaced the complete extension with you new build, it works again. 

Any ideas? :) 

![capture](https://user-images.githubusercontent.com/25410164/34950348-389cc140-fa13-11e7-86fe-ffc350c458bc.JPG)
